ansible/group_vars/nimbus.prater.yml

```diff
@@ -37,53 +37,64 @@ beacon_node_web3_urls: '{{ beacon_node_web3_urls_all | reject("match", "^http://
 
 # Split by hostname for more central location
 nodes_layout:
-  'stable-large-01.aws-eu-central-1a.nimbus.prater':
-    - { branch: 'stable',           start:     0, end:  2500 }
+  'stable-large-01.aws-eu-central-1a.nimbus.prater': # 2000 each
```

Comment:
comment that these are public bootnodes as well - they need a fixed up etc..

Reply:
if it is these - don't remember which ones it was, in the end

Reply:
Yes, those should be the AWS ones, I'll add a comment.